src/renderer.js

```diff
@@ -0,0 +1,21 @@
+//const $ = require('jquery'); // eslint-disable-line no-unused-vars
```

Comment:
I don't know what this line was supposed to do, but it was raising an error. I added jquery. It works fine without it